src/components/views/dialogs/RoomSettingsDialog.js

```diff
@@ -60,9 +61,12 @@ export default class RoomSettingsDialog extends React.Component {
     render() {
         const BaseDialog = sdk.getComponent('views.dialogs.BaseDialog');
 
+        const room = MatrixClientPeg.get().getRoom(this.props.roomId);
+        const title = _t("Room Settings") + ' - ' + room.name;
```

Comment:
This should probably be `_t("Room Settings - %(roomName)s", { roomName: room.name })` because translations